clustering/common/src/main/java/org/jboss/as/clustering/controller/ResourceDescriptor.java

```diff
@@ -172,7 +170,11 @@ public ResourceDescriptor addCapabilities(Predicate<ModelNode> predicate, Collec
     }
 
     public <E extends Enum<E> & ResourceDefinition> ResourceDescriptor addRequiredChildren(Class<E> enumClass) {
-        EnumSet.allOf(enumClass).forEach(definition -> this.requiredChildren.add(definition.getPathElement()));
+        return this.addRequiredChildren(EnumSet.allOf(enumClass));
+    }
+
+    public ResourceDescriptor addRequiredChildren(Set<? extends ResourceDefinition> set) {
+        set.forEach(definition -> this.requiredChildren.add(definition.getPathElement()));
```

Comment:
Use normal for-each loop instead of lambda.

Reply:
Done.